superset/models/helpers.py

```diff
@@ -51,6 +53,10 @@ def json_to_dict(json_str: str) -> Dict[Any, Any]:
 
 
 class ImportMixin:
+    uuid = sa.Column(
```

Comment:
Probably out of scope of this PR, but I think we should rename `ImportMixin` to `ExportImportMixin` eventually

Reply:
Right, I'm planning to do it once we start working on the actual import/export refactoring.